client/src/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface Recipe {
+  id: string;
+  url: string;
+  title: string;
+  notes?: string;
+  images?: string[];
```

Comment:
```suggestion
  images: string[];
```

No need for this to be optional since we can pass an empty array